### PR TITLE
Update default image tag to v1.2.1

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: helm/chaos-mesh/Chart.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-appVersion: "v1.2.0"
+appVersion: "v1.2.1"
 description: Chaos Mesh® is a cloud-native Chaos Engineering platform that orchestrates chaos on Kubernetes environments.
 home: https://chaos-mesh.org
 icon: https://raw.githubusercontent.com/pingcap/chaos-mesh/master/static/logo.svg
 sources:
   - https://github.com/pingcap/chaos-mesh
 name: chaos-mesh
-version: v0.5.0
+version: v0.5.1
```
Comment (STRRL) [inline helm/chaos-mesh/Chart.yaml:9]: The definition of helm files has not updated, do we need to bump this version?

Comment (cwen0) [reply helm/chaos-mesh/Chart.yaml:9]: This [PR](https://github.com/chaos-mesh/chaos-mesh/pull/1867) fixed an issue related to the helm files.

Comment (STRRL) [reply helm/chaos-mesh/Chart.yaml:9]: Sorry I missed that https://github.com/chaos-mesh/chaos-mesh/pull/1867 which includes updates with the helm.

**Review by STRRL — APPROVED** [review]
LGTM

**Review by g1eny0ung — APPROVED** [review]

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**